pyproject.toml

```diff
@@ -82,6 +83,7 @@ pre-commit = {version = "^3.2.1", python = ">=3.8,<4.0"}
 selenium = "^4.11.0"
 types-tabulate = "^0.9.0.3"
 pytest-benchmark = "^4.0.0"
+sqlalchemy = "^2.0.23"
```

Comment:
got a pyright error about connection not having a commit() function, but this makes it go away

Reply:
removing it seems to not cause a problem. I guess local system pre-commit probably doesn't sync up with poetry?